app/index.js

```diff
@@ -12,7 +12,7 @@ const client = new smartcar.AuthClient({
   clientId: process.env.SMARTCAR_CLIENT_ID,
   clientSecret: process.env.SMARTCAR_CLIENT_SECRET,
   redirectUri: process.env.SMARTCAR_REDIRECT_URI,
-  testMode: true,
+  mode: "test",
```

Comment:
```suggestion
  mode: 'test',
```
consistency with rest of file